### DISCUSSION TITLE
Force coefficient curve

### DISCUSSION THREAD

**Discussion by wWL2771140158** [xcompact3d/Incompact3d#161] [Q&A] [answered]
Hello, when I simulate the flow around a hyperbolic cylinder, the calculated lift coefficient curve is not smooth, and there are many jagged oscillations. My calculation domain size is 20Dx10Dx3D, the grid division is 577x289x86, and the time step is t=0.002. What is the reason for the unsmooth force coefficient curve? Is the computational domain not large enough, the grid is too sparse, or the time step is too long, or what other reasons? I hope you can give me some guidance and suggestions.
Below is the lift coefficient diagram I got.
![image](https://user-images.githubusercontent.com/103632822/224521648-f1159de8-6758-446a-90ea-6e57d2206b6a.png)

**Comment by slaizet** [comment, marked as answer]
Hi,
It can be any of your suggestions! I would start with domain size in Y and mesh resolution. 10D is quite small. Depending on the Reynolds number, mesh resolution might be an issue as well.
Best,
Sylvain
